ms-ledger/Dockerfile

```diff
@@ -35,9 +35,9 @@ LABEL license='SPDX-License-Identifier: BSD-3-Clause' \
 RUN apk --no-cache add zeromq
 COPY --from=builder /usr/local/bin/ms-ledger/res/configuration.toml /res/configuration.toml
 COPY --from=builder /usr/local/bin/ms-ledger/main /ms-ledger
-COPY --from=builder /usr/local/bin/ms-ledger/ledger.json /ledger.json
+COPY --from=builder /usr/local/bin/ms-ledger/ledger.json /tmp/ledger.json
```

Comment:
why we want to put the database file under /tmp, as /tmp doesn't sustain in some linux os

Reply:
I wasnt aware that linux will erase the tmp directory periodically. I have since changed the path to "/ledger_db/ledger.json"

Reply:
as mentioned in the meeting with brian, I changed this back to tmp